google-api-grpc/proto-google-cloud-video-intelligence-v1p2beta1/src/main/proto/google/cloud/videointelligence/v1p2beta1/video_intelligence.proto

```diff
@@ -372,15 +379,8 @@ message TextAnnotation {
   // The detected text.
   string text = 1;
 
-  // Confidence for the track of detected text. It is calculated as the highest
-  // over all frames where OCR detected text appears.
-  float confidence = 2;
-
-  // Information related to the frames where OCR detected text appears.
-  repeated TextFrame frames = 3;
-
   // All video segments where OCR detected text appears.
-  repeated TextSegment segments = 4;
+  repeated TextSegment segments = 2;
```

Comment:
This looks like a breaking proto change. Need to verify that it's ok as this is a beta version.

Reply:
This is a beta version and a new beta version _should_ be created. Strictly speaking, this is ok.